developer-docs-site/docs/concepts/coin-and-token/token_airdrop/Token Airdrop Example c4d35e65f7314699a5335fb96d2d5c1f.md

```diff
@@ -0,0 +1,29 @@
+# Token Airdrop Example
+
+We support different ways of transferring token in our token standard [[link](https://aptos.dev/concepts/coin-and-token/aptos-token#token-transfer)]
+
+With the 2 step token transfer, airdropping NFTs to the a set of address can be very straight-ward. 
+
+After minting a token or using existing token from TokenStore, we can directly offer the token to an receiver address from a whitelisted lists. Compared to the NFT mint [example](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/2-Using-Resource-Account/sources/create_nft_with_resource_account.move), we can change the `mint` function to `airdrop` function as below.  It can directly offer the token to a list of addresses 
+
+```rust
+
+public entry fun airdrop() acquires ModuleData {
+    let module_data = borrow_global_mut<ModuleData>(@mint_nft);
+
+    let count = big_vector::length(&whitelisted_addresses);
```

Comment:
where is `whitelisted_addresses` coming from?

Reply:
updated